cat-backup/templates/backup.sh.j2

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+
+NOW=$(date)
+echo "Starting backup at ${NOW}"
+
+echo "Starting database upload to s3..."
+aws --no-progress s3 cp "~/CAT-addresses/db/*.sqlite" "s3://{{ blockchain_backup_bucket }}/{{ network }}/cat-db.sqlite"
```

Comment:
is whatever is generating this file still running? might end up with corrupt backups if this runs in the middle of the file changing.

Reply:
Fixed this